src/store/model/bol.ts

```diff
@@ -0,0 +1,43 @@
+import {Store} from './store';
+
+export const Bol: Store = {
+	labels: {
+		captcha: {
+			container: 'body',
+			text: ['voer de karakters in die u hieronder ziet']
+		},
+		inStock: {
+			container: '.buy-block',
+			text: ['Op voorraad']
```

Comment:
```suggestion
			container: 'div[data-test="buy-block__options"]',
			text: ['In winkelwagen']
```

Isn't this the correct verbiage?